requirements.in

```diff
@@ -7,22 +7,19 @@ django-reversion
 django-cors-headers
 django-filter
 pillow
-psycopg2>=2.9
+psycopg[c]
 easy-thumbnails
 drf-nested-routers
-drf-oidc-auth<1
+drf-oidc-auth
```

Comment:
Is `drf-oidc-auth` still required?

Reply:
Nope, this PR has been sitting here a long time and then it was not in the removal list. There's things to go through here.